spec/features/idv/in_person_spec.rb

```diff
@@ -269,6 +347,74 @@
       perform_desktop_hybrid_steps(user, same_address_as_id: false)
     end
 
+    context 'when the user fails docauth remote in the hybrid flow and begins IPP' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled).and_return(true)
+
+        perform_in_browser(:desktop) do
+          visit_idp_from_sp_with_ial2(:oidc, **{ client_id: ipp_service_provider.issuer })
+          sign_in_via_branded_page(user)
+          complete_doc_auth_steps_before_hybrid_handoff_step
+
+          # choose remote
+          click_on t('forms.buttons.continue_remote')
+          click_send_link
+
+          expect(page).to have_content(t('doc_auth.headings.text_message'))
+        end
+
+        perform_mobile_hybrid_steps
+      end
+
+      context 'then the user navigates to the how to verify page and changes from IPP to remote
+      verification after returning to desktop' do
+        it 'allows the user to successfully complete remote identity verification' do
+          perform_in_browser(:desktop) do
+            # Change mind and resume remote identity verification
+            visit idv_how_to_verify_url
+
+            # choose remote
+            click_on t('forms.buttons.continue_remote')
+            complete_hybrid_handoff_step
+            successful_response = instance_double(
+              Faraday::Response,
+              status: 200,
+              body: LexisNexisFixtures.true_id_response_success,
+            )
+            DocAuth::Mock::DocAuthMockClient.mock_response!(
+              method: :get_results,
+              response: DocAuth::LexisNexis::Responses::TrueIdResponse.new(
+                successful_response,
+                DocAuth::LexisNexis::Config.new,
+              ),
+            )
+            complete_document_capture_step(with_selfie: false)
+
+            complete_remote_idv_from_ssn(user)
+          end
+        end
+      end
+
+      context 'then navigates back to the hybrid handoff page and selects remote verification
+      via the hybrid flow' do
+        it 'allows the user to successfully complete remote identity verification',
+           allow_browser_log: true do
+          # click back link while on the state id page
+          perform_in_browser(:desktop) do
+            visit idv_hybrid_handoff_url
+            click_send_link
+
+            # Test that user stays on the link sent page
+            sleep(5)
```

Comment:
I added a `sleep` here because when we reproduced the bug, the user didn't immediately get moved from the link sent page to the state id page.  If I added a `wait` to the expect statement, there would have been a risk of a false positive.